recipes-core/base-files/files/fstab_selinux.append

```diff
@@ -0,0 +1,13 @@
+# stock fstab - you probably want to override this with a machine specific one
+
+/dev/root            /                    auto       ro              1  0
+proc                 /proc                proc       defaults              0  0
+devpts               /dev/pts             devpts     mode=0620,ptmxmode=0666,gid=5      0  0
+
+/dev/hda4            /home                ext4       defaults,noatime       0  0
+/dev/hda5            /var                 ext4       defaults,noatime       0  0
+
+/dev/aosvg/downloads /var/aos/downloads  ext4     defaults,nofail,noatime,x-systemd.device-timeout=5s,context="system_u:object_r:aos_var_run_t:s0"  0  0
```

Comment:
You don't need quotas for context parma, do you?

Reply:
Yes, you are right. Removed